docs/content/guides/dagster/pythonic-resources.mdx

```diff
@@ -111,6 +111,121 @@ defs = Definitions(
 </TabItem>
 </TabGroup>
 
+### Using resources with sensors and schedules
```

Comment:
i think it's worth splitting sensors and schedules into two sections, as opposed to tabs. cos i don't think they are usually mutually exclusive or pick one over the other. benefit of having both flattened out in the page is that i can simply `cmd-F` and find sensor or schedule (which was what i did when i was trying to find this doc)

Reply:
I agree that this isn't an either-or situation, so tabs might not be the best fit.

Reply:
Will leave both as top-level sections!